build.gradle

```
@@ -98,9 +98,9 @@ ext {
 
   testing = [
           junit             : 'junit:junit:4.13-beta-3',
-          robolectric       : 'org.robolectric:robolectric:4.3',
+          robolectric       : 'org.robolectric:robolectric:4.3.1',
           privateConstructor: 'com.pushtorefresh.java-private-constructor-checker:checker:1.2.0',
-          assertJ           : 'org.assertj:assertj-core:2.9.1'
+          assertJ           : 'org.assertj:assertj-core:3.13.2'
```

```suggestion
          assertJ           : 'org.assertj:assertj-core:2.9.1'
```

3.x is java 8 and we don't have that fully